__tests__/Unit/hooks/tasksApi.test.tsx

```diff
@@ -0,0 +1,212 @@
+            type: 'feature',
+            status: 'AVAILABLE',
+            percentCompleted: 0,
+            priority: 'TBD',
+            github: {
+                issue: {
+                    status: 'open',
+                    id: 697061944,
+                },
+            },
+        };
+        const { result, waitForNextUpdate } = renderHook(
+            () => useAddTaskMutation(),
+            {
+                wrapper: Wrapper,
+            }
+        );
+        const [addTask, initialResponse] = result.current;
+        expect(initialResponse.isLoading).toBe(false);
+        expect(initialResponse.data).toBeUndefined();
+
+        act(() => {
+            addTask(newTask);
+        });
+
+        const loadingResponse = result.current[1];
+        expect(loadingResponse.isLoading).toBe(true);
+
+        await act(() => waitForNextUpdate());
+
+        const nextResponse = result.current[1];
+        expect(nextResponse.data).not.toBeUndefined();
+        expect(nextResponse.data?.message).toBe('Task created successfully!');
+        expect(nextResponse.data?.task).not.toBeUndefined();
+        expect(nextResponse.data?.task).toStrictEqual(newTask);
+    });
+
+    test('should fail to create a new task with error', async () => {
+        server.use(failedAddNewTaskHandler);
+        const { result, waitForNextUpdate } = renderHook(
+            () => useAddTaskMutation(),
+            {
+                wrapper: Wrapper,
+            }
+        );
+        const [addTask, initialResponse] = result.current;
+        expect(initialResponse.isLoading).toBe(false);
+        expect(initialResponse.data).toBeUndefined();
+
+        act(() => {
+            addTask({
+                type: 'feature',
+                status: 'AVAILABLE',
+                percentCompleted: 0,
+            });
+        });
+
+        await act(() => waitForNextUpdate());
+
+        const nextResponse = result.current[1];
+        expect(nextResponse.isError).toBe(true);
+        expect(nextResponse.error).toHaveProperty('status', 404);
+        expect(nextResponse.error).toHaveProperty('data', {
+            statusCode: 400,
+            error: 'Bad Request',
+            message: '"title" is required',
+        });
+    });
+});
+
+describe('useUpdateTaskMutation()', () => {
+    const updatedTaskData = {
+        status: 'COMPLETED',
+    };
+
+    test('updates a task', async () => {
+        const taskId = '1eJhUW19D556AhPEpdPr';
+        const { result, waitForNextUpdate } = renderHook(
+            () => useUpdateTaskMutation(),
+            {
+                wrapper: Wrapper,
+            }
+        );
+        const [updateTask, initialResponse] = result.current;
+        expect(initialResponse.isLoading).toBe(false);
+        expect(initialResponse.data).toBeUndefined();
+
+        act(() => {
+            updateTask({
+                task: updatedTaskData,
+                id: taskId,
+            });
```

Comment:
create a constant and then pass it to the function

Reply:
Done